denizen_scripts/survival/repo-link/cuboid_creator.dsc

```diff
@@ -45,9 +45,19 @@ cuboid_command:
         - if <player.has_flag[point1]> && <player.has_flag[point2]>:
           - note <cuboid[<player.flag[point1]>|<player.flag[point2]>]> as:<context.args.get[2]>
```

Comment:
a note for later, this cuboid notation: `<cuboid[<LocationTag>|<LocationTag>]>` should be converted to:
`<LocationTag.to_cuboid[<[LocationTag]>]>`

Reply:
zero == 0